src/app/frontend/common/filters/cores_filter.js

```diff
@@ -39,7 +42,7 @@ export default function coresFilter(numberFilter) {
       power += 1;
     }
     let formatted = numberFilter(value / divider);
-    return `${formatted} ${powerSuffixes[power]}CPU`;
+    return `${formatted} ${powerSuffixes[power]}`;
```

Comment:
The trailing space in the case of `{{1| kdCores}}` might surprise people using the filter - would you consider the slightly uglier

```js
    let suffix = powerSuffixes[power];
    return  suffix ? `${formatted} ${suffix} : formatted;
```